build.gradle.kts

```diff
@@ -68,7 +68,7 @@ kotlin {
         val androidMain by getting {
             dependencies {
                 rootProject
-                implementation("org.pytorch:pytorch_android_lite:1.11")
+                implementation("org.pytorch:pytorch_android_lite:1.12")
```

Comment:
there is a `1.12.1` version

Reply:
right, looks like `1.12.1` is the way to go for now